spec/book_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../book'
+
+describe Book do
+  before(:each) do
+    @book = Book.new('Hello', 'Ruby')
+  end
+  it 'create book title' do
+    expect(@book.title).to eq 'Hello'
+  end
+
+  it 'create book author' do
+    expect(@book.author).to eq 'Ruby'
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_rental` classbook method has (at least) an 'it' block 👇🏻 check your book.rb file

```ruby
   def add_rental(person, date)
      Rental.new(date, person, self)
   end
```

Reply:
Already implemented it! Thanks for the feedback! :smile: